src/apps/ENGINE/src/compiler.cpp

```diff
@@ -6626,7 +6646,6 @@ bool COMPILER::LoadState(std::fstream &fileS)
             SetError("missing variable name");
             return false;
         }
-        Assert(utf8::IsValidUtf8(pString));
```

Comment:
Should we log this behaviour? It may help detecting non-converted files

Reply:
since now there is a check in the `ReadString` function, no need to have the same check here. I removed it.